public/js/eventsPage.js

```diff
@@ -46,20 +46,21 @@ function displayEventsHandler(eventsAndHiringPartners) {
     } else {
         eventList.innerHTML = ''
         displayEvents(eventsAndHiringPartners.events.data, eventsAndHiringPartners.hiringPartners).then(() => {
-            let showInfoButtons = document.querySelectorAll('.show-event-info')
-                    showInfoButtons.forEach(function (button) {
-                        button.addEventListener('click', e => {
-                            let targetId = 'moreInfo' + e.target.dataset.reference
-                            let targetDiv = document.getElementById(targetId)
-                            targetDiv.classList.toggle('hidden')
-                            targetDiv.parentElement.classList.toggle('open')
+            let showInfoHeaders = document.querySelectorAll('.header-show-event-info')
+            showInfoHeaders.forEach(function (header) {
+                header.addEventListener('click', e => {
+                    let buttonId = 'button' + e.target.dataset.reference
+                    let targetId = 'moreInfo' + e.target.dataset.reference
+                    let targetDiv = document.getElementById(targetId)
```

Comment:
query selector

Reply:
Discussed with you that this is code we copied  and then tweaked(from another area of the project) and you're happy for us not to alter it :)